pytube/__main__.py

```diff
@@ -102,6 +103,34 @@ def __init__(
             self.prefetch()
             self.descramble()
 
+    def check_availability(self):
```

Comment:
nit: check_availability -> check_video_availability ?

Reply:
I don't think we really need to use a more verbose function name here. This function should never be called by a consumer of the library, and we don't need to make the code look like java to be readable :p

Reply:
I'm hesitant to make long, wordy function names because of the fact that we have to take line length into account elsewhere in the code. I don't know if it's the case here, but I have had to split function calls over multiple lines thanks to function and variable names simply being too long, and that significantly detracts from readability imo